pkg/maps/cidrmap/cidrmap.go

```diff
@@ -100,28 +154,40 @@ func (cm *CIDRMap) Close() error {
 
 // OpenMap opens a new CIDRMap. 'bool' returns 'true' if the map was
 // created, and 'false' if the map already existed.
-func OpenMap(path string, prefixlen int) (*CIDRMap, bool, error) {
+func OpenMap(path string, prefixlen int, prefixdyn bool) (*CIDRMap, bool, error) {
```

Comment:
Can you add a comment on what's `prefixdyn`?

Reply:
Fixed.